aws_change_observer/aws_change_observer_stack.py

```diff
@@ -109,3 +114,37 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
              allow_origins=apigateway.Cors.ALL_ORIGINS,
              allow_methods=["GET", "POST", "OPTIONS"],
         )
+
+        if is_prod:
```

Comment:
@PCBush @miitchellstahl @murauskas101 
API Gateway URL is linked to `change-observer.com` only if is_prod is set to True. For development on your account set it to False